src/tests/blueprint/t_blueprint_mesh_verify.cpp

```diff
@@ -1718,6 +1719,20 @@ TEST(conduit_blueprint_mesh_verify, index_matset)
             CHECK_MESH(verify_matset_index,mindex,info,true);
         }
 
+        if(mindex.has_child("material_maps"))
+        { // Materials Field Tests //
+            mindex.remove("material_map");
+            CHECK_MESH(verify_matset_index,mindex,info,false);
+
+            mindex["material_map"];
+            CHECK_MESH(verify_matset_index,mindex,info,false);
+
+            mindex["material_map/mat1"].set(1);
+            CHECK_MESH(verify_matset_index,mindex,info,true);
+            mindex["material_map/mat2"].set(2);
+            CHECK_MESH(verify_matset_index,mindex,info,true);
+        }
```

Comment:
Great! However, for the sake of completeness, we also need this to be tested at the `matset` level for multi-buffer matsets, and for the documentation to be updated.